Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -358,14 +358,16 @@ class In: Operation {
 
 class BeginAnyFunctionDefinition: Operation {
     let signature: FunctionSignature
-    
+    let isStrict: Bool
+
     /// Whether the last parameter is a rest parameter.
     var hasRestParam: Bool {
         return signature.hasVarargsParameter()
     }
     
-    init(signature: FunctionSignature) {
+    init(signature: FunctionSignature, isStrict: Bool = false) {
```

Comment:
Nit: is the default parameter value necessary? Just because the other Operations don't have default parameters either.